firetweet/src/main/java/org/getlantern/firetweet/model/Lantern.java

```diff
@@ -63,4 +92,37 @@ public static void stop() {
             }
         }
     }
+
+    public static void checkNewVersion() {
+
+        if (context != null) {
+            try {
+
+                // In developer mode; don't check for updates
+                if ((context.getPackageManager().getPackageInfo(
+                        context.getPackageName(), 0).applicationInfo.flags &
+                        ApplicationInfo.FLAG_DEBUGGABLE) != 0) {
+                    Log.d(LOG_TAG, "Not showing update notification in developer mode.");
+                    return;
```

Comment:
@atavism When I tested the auto-update feature, I spent half an hour before realized that it's intentionally skipped in dev mode. I guess it may cause more confusion later than extra update prompt during development.

Ideally, we may compare the dot separated digits in version string.